pkg/kubecost/window.go

```diff
@@ -482,6 +577,47 @@ func (w Window) DurationOffset() (time.Duration, time.Duration, error) {
 	return duration, offset, nil
 }
 
+// DurationOffsetForPrometheus returns durations representing the duration and
+// offset of the given window, factoring in the Thanos offset if necessary. The
+// duration is returned as
```

Comment:
Returned as...?

Reply:
LOL. Good catch, fixed. Let me know if the comment still doesn't make sense for whatever reason.

Reply:
Looks awesome.